dashboard/controller/users/editUser.js

```diff
@@ -22,14 +22,14 @@ module.exports = function editUser(req, res) {
 			req.body.password = req.body.password.trim();
 			req.body.color = JSON.parse(req.body.color);
 			req.assert('name', common.l10n.get('UsernameInvalid')).matches(regexValidate("user"));
-			req.assert('password', common.l10n.get('PasswordAtLeast', {count:users.ini().security.min_password_size})).len(users.ini().security.min_password_size);
+			req.assert('password', common.l10n.get('PasswordAtLeast', { count: users.ini().security.min_password_size })).len(users.ini().security.min_password_size);
```

Comment:
If changes in the code of this file does not affects the Assignment Feature, then reset the changes in this file.

Reply:
The changes to this file are still irrelevant. Please reset the changes made in this file.